src/app/views/users/edit/EditUser.jsx

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useCallback, useState } from "react";
+import { withRouter } from "react-router";
+import isEqual from "lodash/isEqual";
+import isEmpty from "lodash/isEmpty";
+import PropTypes from "prop-types";
+import { Link } from "react-router";
+import url from "../../../utilities/url";
+import Input from "../../../components/Input";
+import Warning from "../../../icons/Warning";
+import validate from "./validate";
+import FormValidationErrors from "../../../components/form-validation-errors";
+import BackButton from "../../../components/back-button";
+import FormFooter from "../../../components/form-footer";
+import SelectedItemList from "../../../components/selected-items/SelectedItemList";
+import Loader from "../../../components/loader/index";
+import RadioGroup from "../../../components/radio-buttons/RadioGroup";
+import TextArea from "../../../components/text-area/TextArea";
+
+const options = [
```

Comment:
Not a required change but a note, the name `options` is a bit generic, another developer who may need to maintain this in the future will not easily know what 'options' these relate to.